meorg_client/client.py

```diff
@@ -25,84 +26,128 @@ def __init__(self, base_url=mcc.MEORG_URL, email=None, password=None):
         password : str, optional
             User password, by default None
         """
-        self.base_url = base_url
-        self.headers = dict()
+
+        # Initialise the mimetypes
+        mt.init()
+
+        # Ensure there is a trailing slash
+        self.base_url = f"{base_url}/" if base_url[-1] != "/" else base_url
```

Comment:
The passed on `base_url` could be empty. An initial check, or using `endswith()` could be some ways.

Reply:
Minor typo in the docstring - Supplying instead of Suppying

Reply:
I think a better solution would be to explicitly set the base URL inside the class and set a dev argument. I don't see a situation where the URL is likely to change and if it does we will need to make internal changes anyway.

Will change.